README.md

```diff
@@ -47,6 +47,7 @@ An example `.envrc` is provided for optional but recommended use with [`direnv`]
 * `OTEL_EXPORTER_PROMETHEUS_HOST` -- default `"localhost"`
 * `OTEL_EXPORTER_PROMETHEUS_PORT` -- default `9464`
 * `SKIP_APP_COMPILATION` -- we currently build the react app in a build script. To avoid this behavior, set this environment variable.
+* `ASSET_DIR` -- set this to skip building the react app and include react app assets from a different directory
```

Comment:
Is it useful to have two different env vars for this? The docs for ASSET_DIR and SKIP_APP_COMPILATION sound very similar. In #67 I made it so `SKIP_APP_COMPILATION=1` doesn't even serve the assets, as I use that for local development when I'm running `npm start`.

Reply:
I think so, my thinking was that SKIP_APP_COMPILATION is worth keeping because of its ease of use, since you don't need to provide a valid path. With #67's changes, I think the case for separate variables is stronger.